pageserver/src/billing_metrics.rs

```diff
@@ -32,16 +35,23 @@ use reqwest::Url;
 /// "tenant_id": "5d07d9ce9237c4cd845ea7918c0afa7d",
 /// "timeline_id": "00000000000000000000000000000000",
 /// "time": ...,
+/// "idempotency_key":
```

Comment:
NIT: Should there be an example as it is for other fields?

Reply:
fixed